src/serac/physics/tests/beam_bending.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2019-2023, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "serac/physics/solid_mechanics.hpp"
+
+#include <functional>
+#include <fstream>
+#include <set>
+#include <string>
+
+#include "axom/slic/core/SimpleLogger.hpp"
+#include <gtest/gtest.h>
+#include "mfem.hpp"
+
+#include "serac/mesh/mesh_utils.hpp"
+#include "serac/physics/state/state_manager.hpp"
+#include "serac/physics/materials/solid_material.hpp"
+#include "serac/serac_config.hpp"
+
+namespace serac {
+
+TEST(BeamBending, TwoDimensional)
+{
+  constexpr int p   = 1;
+  constexpr int dim = 2;
+
+  MPI_Barrier(MPI_COMM_WORLD);
+
+  // Create DataStore
+  axom::sidre::DataStore datastore;
+  serac::StateManager::initialize(datastore, "beam_bending_data");
+
+  // Construct the appropriate dimension mesh and give it to the data store
+  std::string filename = SERAC_REPO_DIR "/data/meshes/beam-quad.mesh";
+
+  auto mesh = mesh::refineAndDistribute(buildMeshFromFile(filename), 0, 0);
+  serac::StateManager::setMesh(std::move(mesh));
+
+  // _solver_params_start
```

Comment:
I believe these are copied from the other example that I pull into the docs page. They should be deleted here.